docs/resources/metric_dashboard.md

```diff
@@ -79,61 +80,88 @@ resource "lightstep_metric_dashboard" "customer_charges" {
 
 ### Required
 
-- **dashboard_name** (String)
-- **project_name** (String)
+- `dashboard_name` (String)
+- `project_name` (String)
 
 ### Optional
 
-- **chart** (Block List) (see [below for nested schema](#nestedblock--chart))
-- **id** (String) The ID of this resource.
+- `chart` (Block Set) (see [below for nested schema](#nestedblock--chart))
 
 ### Read-Only
 
-- **type** (String)
+- `id` (String) The ID of this resource.
+- `type` (String)
 
 <a id="nestedblock--chart"></a>
 ### Nested Schema for `chart`
 
 Required:
 
-- **name** (String)
-- **query** (Block List, Min: 1) (see [below for nested schema](#nestedblock--chart--query))
-- **rank** (Number)
-- **type** (String)
+- `name` (String)
+- `query` (Block List, Min: 1) (see [below for nested schema](#nestedblock--chart--query))
+- `rank` (Number)
+- `type` (String)
 
 Optional:
 
-- **y_axis** (Block List, Max: 1) (see [below for nested schema](#nestedblock--chart--y_axis))
+- `y_axis` (Block List, Max: 1, Deprecated) (see [below for nested schema](#nestedblock--chart--y_axis))
 
 Read-Only:
 
-- **id** (String) The ID of this resource.
+- `id` (String) The ID of this resource.
 
 <a id="nestedblock--chart--query"></a>
 ### Nested Schema for `chart.query`
 
 Required:
 
-- **hidden** (Boolean)
-- **query_name** (String)
+- `hidden` (Boolean)
+- `query_name` (String)
+
+Optional:
+
+- `display` (String)
+- `exclude_filters` (List of Map of String) Not-equals filters (operand: neq)
+- `filters` (List of Map of String) Non-equality filters (operand: contains, regexp)
```

Comment:
Is this field new? It seems unrelated to the rest of your changes, and potentially redundant with `include_filters`?

Reply:
I did not change anything with filters, include_filters, etc. I just don't think the docs have been updated in a while, so the `tfplugindocs` command added it here. 

BTW, AFAICT `filters` is used for regex matching only.